src/Controller/MemberController.php

```diff
@@ -75,13 +75,19 @@ public function apply(Request $request): Response {
 
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
-            $em = $this->getDoctrine()->getManager();
-            $em->persist($member);
-            $em->flush();
-
-            return $this->render('user/apply.html.twig', [
-                'success' => true
-            ]);
+            $memberRepository = $this->getDoctrine()->getRepository(Member::class);
+            $existingMember = $memberRepository->findOneByEmail($member->getEmail());
+            if ($existingMember !== null) {
+                $form->addError(new FormError('Er is al een lid met dit e-mailadres.'));
```

Comment:
Laten we dát dus niet doen. Normale flow moet zijn:
1. zegt dat een verificatie-email is verstuurt
2. Jij accepteert het als je toegang hebt tot de email, of anders zie je het niet
Dat houdt het privacy-veilig